darts/models/forecasting_model.py

```diff
@@ -700,19 +719,17 @@ def fit(self,
         Parameters
         ----------
         series
-            A time series. The model will be trained to forecast this time series.
+            The model will be trained to forecast this time series. Can be multivariate.
```

Comment:
Whether it can be multivariate or not depends on the models implementations. for instance ARIMA doesn't support multivariate (VARIMA does).

Reply:
I adapted the description